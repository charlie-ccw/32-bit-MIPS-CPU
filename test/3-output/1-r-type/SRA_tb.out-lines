register_v0_final f3df40da
