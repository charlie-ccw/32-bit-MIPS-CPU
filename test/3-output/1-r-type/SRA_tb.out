f3df40da
